lib/findbugs_translate_checkstyle_format/version.rb

```diff
@@ -1,3 +1,3 @@
 module FindbugsTranslateCheckstyleFormat
-  VERSION = '0.2.6'
+  VERSION = '0.3.0'
```

Comment:
__[rubocop]__
* Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->